### PR TITLE
2_5 added

### PR DESCRIPTION
**Pull request by maleki1999**
2_5 added

### CODE REVIEW

File: 2_5/Pythagorean_numbers/main.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+contents = []
+count = 0
+while count<3: 
+    contents.append(int(input()))
+    count+=1
+max_list = contents.pop(contents.index(max(contents)))
+sum_list =0 
+for n in contents:
+    sum_list+=n**2 
+print("YES" if max_list**2 == sum_list else "NO" )
```
Comment (abedimhosein) [inline 2_5/Pythagorean_numbers/main.py:10]: @maleki1999 
```
numbers = [int(input()) for _ in range(3)]
c = numbers.pop(numbers.index(max(numbers)))
a, b = numbers
print('YES') if c ** 2 == a ** 2 + b ** 2 else print('NO')
```

File: 2_5/multiplication/main.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+n = int(input())
+arr = list(range(1,n+1))
+for i in range(1 , n+1):
+    print(' '.join(map(str, [element * i for element in arr])))
```
Comment (abedimhosein) [inline 2_5/multiplication/main.py:4]: @maleki1999 
think simple

File: 2_5/single digit/main.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+number = [int(digit) for digit in input()]
+while len(number)>1:
+    number = [int(digit) for digit in str(sum(number))]
+print(number[0])
```
Comment (abedimhosein) [inline 2_5/single digit/main.py:4]: @maleki1999 
Not a bad idea but, Write this code using recursive functions!!!!!!!!
and don't merge this pull request until you write this properly

Comment (maleki1999) [reply 2_5/single digit/main.py:4]: def sum_digit(number):
    sum = 0
    for digit in number:
        sum+=int(digit)
    return str(sum)
        
num = input()
while len(num)>1:
    num = sum_digit(num)
print(num)

Comment (maleki1999) [reply 2_5/single digit/main.py:4]: [main2.txt](https://github.com/maleki1999/QueraEasyQ/files/9226752/main2.txt)

Comment (maleki1999) [reply 2_5/single digit/main.py:4]: please check this code.
@abedimhosein

Comment (abedimhosein) [reply 2_5/single digit/main.py:4]: @maleki1999 
my solution
```
def make_it_one_digit(number: str):
    if len(number) == 1:
        return int(number)

    number = str(sum(int(i) for i in number))
    return make_it_one_digit(number)

print(make_it_one_digit(input()))
```

Comment (abedimhosein) [reply 2_5/single digit/main.py:4]: @maleki1999 
merge this

**Pull request merged**